scripts/calculateRevenue/calculateRevenue.ts

```diff
@@ -10,22 +10,31 @@ async function main(args: ReturnType<typeof parseArgs>) {
 
   const handler = calculateRevenueHandlers[args['protocol-id'] as Protocol]
 
-  const allResults: Record<string, RevenueResult> = {}
+  const allResults: Array<{
+    address: string
+    revenue: number
+  }> = []
 
   for (let i = 0; i < eligibleAddresses.length; i++) {
     const address = eligibleAddresses[i]
     console.log(
       `Calculating revenue for ${address} (${i + 1}/${eligibleAddresses.length})`,
     )
-    const userResult = await handler({
+    const revenue = await handler({
       address,
       startTimestamp: new Date(args['start-timestamp']),
       endTimestamp: new Date(args['end-timestamp']),
     })
-    allResults[address] = userResult
+    allResults.push({
+      address,
+      revenue,
+    })
   }
+  const output = allResults
+    .map((result) => `${result.address},${result.revenue}`)
+    .join('\n')
 
-  writeFileSync(args['output-file'], JSON.stringify(allResults, null, 4))
+  writeFileSync(args['output-file'], output)
```

Comment:
Shouldn't matter, but to be safe.
```suggestion
  writeFileSync(args['output-file'], output, { encoding: 'utf-8' })
```